src/trios/TRIOS_Domain.H

```diff
@@ -284,7 +293,16 @@ namespace TRIOS {
         //! we also offer this option for matrices, the others are not so important
         int Standard2Solve(const Epetra_CrsMatrix& source, Epetra_CrsMatrix& target) const;
 
+        //! vertical grid stretching function
```

Comment:
I'd prefer these not to be public. If they really have to be, please turn them into standard methods. If you just need this for the tests, just implement a class in the tests that inherits from this class and has a way to access these.

Reply:
Main reason is that several things need values derived from these and the `Z` position, so the alternative is to add methods/accessors for each derived value too. I don't have a particular objection to adding those, I was just lazy for the initial version.

Reply:
Since this has to communicate with something outside of I-EMIC it's good to think about the right interface first so you don't have to change things on both side all the time.